src/games/strategy/common/ui/BasicGameMenuBar.java

```diff
@@ -413,7 +413,7 @@ public void actionPerformed(final ActionEvent e) {
 
   protected void addAboutMenu(final JMenu parentMenu) {
     final String text = "<h2>" + getData().getGameName() + "</h2>" + "<p><b>Engine Version:</b> "
-        + games.strategy.engine.EngineVersion.VERSION.toString() + "<br><b>Game:</b> " + getData().getGameName()
+        + games.strategy.engine.ClientContext.getInstance().engineVersion().getVersion().toString() + "<br><b>Game:</b> " + getData().getGameName()
```

Comment:
This update will be around a lot, changing client code to get the EngineVersion not through `EngineVersion.VERSION`, but instead `ClientContext.getIntance().engineVersion()`

ClientContext has a pattern going where the name of the object you want is the name of the method. I considered naming things as "getObject()" instead of "object()" , but in the context above, since you almost always have the "getInstance()" call, it's a whole lot of "gets" in one line. What then convinced me was nice symmetry in ClientContext from this change, things were pretty simple there and well related by names. That is important since that class will grow a lot, and if it doesn't start dead simple then we'll be in trouble.